extension/src/background/helpers/account.ts

```diff
@@ -164,6 +164,22 @@ export const getIsRpcHealthy = async (networkDetails: NetworkDetails) => {
   return rpcHealth.status === "healthy";
 };
 
+export const getUserNotification = async () => {
+  let response = { enabled: false, message: "" };
+
+  try {
+    const res = await fetch(`${INDEXER_URL}/user-notification`);
+    response = await res.json();
+  } catch (e) {
+    captureException(
+      `Failed to load feature flag for Soroban mainnet - ${JSON.stringify(e)}`,
```

Comment:
Nit but feature flags are not per network right now

Reply:
Oh good catch - bad copy/paste job